lms/djangoapps/learner_dashboard/tests/test_utils.py

```diff
@@ -21,3 +28,40 @@ def test_strip_course_id(self, path):
         """
         actual = utils.strip_course_id(path + unicode(utils.FAKE_COURSE_KEY))
         self.assertEqual(actual, path)
+
+
+class DisclaimerIncompleteFieldsTestCase(TestCase):
+    """
+    Test cases to display or not the reminder alert if a user
+    has left some required fields in blank after n days.
+    """
+
+    def setUp(self):
+        self.user = User.objects.create(
+            username='my_self_user',
+            date_joined=date(2016, 03, 01)
```

Comment:
Also, can we add another 2 test cases where `date_joined` is below and above the threshold, thus leading to showing/not-showing alerts based off of `date_joined` rather than just the filled in fields?